eval/bedrock_runner.py

```diff
@@ -38,20 +38,10 @@ def process_row(row, model_id, decimal_points):
     generated_query = model_response["generation"]
     end_time = time()
 
-    if "```sql" in generated_query:
-        generated_query = (
-            generated_query.split("[/SQL]")[0]
-            .split("```sql")[-1]
-            .split("```")[0]
-            .split(";")[0]
-            .strip()
-            + ";"
-        )
-    else:
-        generated_query = (
-            generated_query.split("[/SQL]")[0].split("```")[1].split(";")[0].strip()
-            + ";"
-        )
+    generated_query = (
+        generated_query.split("```sql")[-1].split("```")[0].split(";")[0].strip() + ";"
+    )
+    print(generated_query)
```

Comment:
nit: should we remove the print statement?

Reply:
Will do!